mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule PhoenixSwagger.Mixfile do
   use Mix.Project
 
-  @version "0.6.4"
+  @version "0.6.5"
```

Comment:
We're pre 1.0 so we usually introduce breaking changes by bumping the minor version, and backwards compatible changes with the patch version.

I would prefer this be a minor version bump, since it is a breaking change to anyone that adopted the phoenix 1.3-rc convention for module names. Even if those users update to phoenix 1.3, they may choose to not rename their modules straight away.

However, maybe I'm over-thinking this - do we really need to worry about users that adopted the unreleased phoenix version?

Ping @surik to weigh in here.